test/TopupEUReTest.t.sol

```diff
@@ -28,13 +28,12 @@ contract TopupTest is BaseFixture {
         uint256 initialBptBal = IERC20(BPT_STEUR_EURE).balanceOf(address(safe));
         uint256 initialEureBal = IERC20(EURE).balanceOf(address(safe));
 
-        (bool canExec, bytes memory execPayload) = roboModule.checker();
-        (bytes memory dataWithoutSelector, bytes4 selector) = _extractEncodeDataWithoutSelector(execPayload);
+        (bool canExec, bytes memory execPayload) = roboModule.checkUpkeep("");
         (RoboSaverVirtualModule.PoolAction _action, uint256 _deficit) =
-            abi.decode(dataWithoutSelector, (RoboSaverVirtualModule.PoolAction, uint256));
+            abi.decode(execPayload, (RoboSaverVirtualModule.PoolAction, uint256));
 
         assertTrue(canExec, "CanExec: not executable");
-        assertEq(selector, ADJUST_POOL_SELECTOR, "Selector: not adjust pool (0xba2f0056)");
+        // assertEq(selector, ADJUST_POOL_SELECTOR, "Selector: not adjust pool (0xba2f0056)");
```

Comment:
idem

Reply:
this one just below has the other assert of the action